### PR TITLE
GL-853 Modify VcfToAdpc to handle multiple samples.

### PR DESCRIPTION
**Pull request by gbggrant**
### Description

Modify VcfToAdpc to handle multiple samples.

Put NaN in the adc.bin file if normalized intensity is null.
Output files containing number of samples and markers (useful for running VerifyIDIntensity itself).
Added tests

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

#### Content
- [ ] Added or modified tests to cover changes and any new functionality
- [ ] Edited the README / documentation (if applicable)
- [ ] All tests passing on Travis

#### Review
- [ ] Final thumbs-up from reviewer
- [ ] Rebase, squash and reword as applicable

For more detailed guidelines, see https://github.com/broadinstitute/picard/wiki/Guidelines-for-pull-requests

### CODE REVIEW

**Comment by gbggrant** [comment]
retest this please

File: src/main/java/picard/arrays/VcfToAdpc.java [hunk 1]
```diff
@@ -62,55 +68,85 @@
...
+    @Argument(shortName = "NMF", doc = "A text file into which the number of loci in the VCF will be written. " +
+            "This is useful for calling verifyIDIntensity.")
+    public File NUM_MARKERS_FILE;
+
     @Override
     protected int doWork() {
-        IOUtil.assertFileIsReadable(VCF);
+        final List<File> inputs = IOUtil.unrollFiles(VCF, IOUtil.VCF_EXTENSIONS);
+        IOUtil.assertFilesAreWritable(inputs);
+        IOUtil.assertFileIsWritable(SAMPLES_FILE);
```
Comment (jessicaway) [inline src/main/java/picard/arrays/VcfToAdpc.java]: Should this also check the NUM_MARKERS_FILE?

File: src/main/java/picard/arrays/VcfToAdpc.java [hunk 2]
```diff
@@ -62,55 +68,85 @@
...
+
+    @Argument(shortName = "NMF", doc = "A text file into which the number of loci in the VCF will be written. " +
+            "This is useful for calling verifyIDIntensity.")
+    public File NUM_MARKERS_FILE;
+
     @Override
     protected int doWork() {
-        IOUtil.assertFileIsReadable(VCF);
+        final List<File> inputs = IOUtil.unrollFiles(VCF, IOUtil.VCF_EXTENSIONS);
+        IOUtil.assertFilesAreWritable(inputs);
```
Comment (jessicaway) [inline src/main/java/picard/arrays/VcfToAdpc.java]: Do we need to check writeable? The previous check was for readable

File: src/main/java/picard/arrays/VcfToAdpc.java (hunk 1 above)
Comment (gbggrant) [reply src/main/java/picard/arrays/VcfToAdpc.java]: Yes.

File: src/main/java/picard/arrays/VcfToAdpc.java (hunk 2 above)
Comment (gbggrant) [reply src/main/java/picard/arrays/VcfToAdpc.java]: Oof.  Yeah, just mistranscribed that.

**Pull request merged**